src/files/extraction/helpers.ts

```diff
@@ -87,6 +97,30 @@ export async function scheduleExtraction(
   backend: ExtractionBackend = EXTRACTION_BACKEND
 ) {
   switch (backend) {
+    case ExtractionBackend.DOCLING: {
+      file.extraction = new DoclingExtraction({ jobId: file.id });
+      await ORM.em.flush();
+      if (file.mimeType?.startsWith('text/') || file.mimeType === 'application/json') {
+        await nodeQueue.add(
+          QueueName.FILES_EXTRACTION_NODE,
```

Comment:
Why is this sent to the node worker when the worker doesn't do any extraction?

Reply:
The worker is doing simple text extraction